CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## unreleased
 
+## v4.5.0 - 2023.01.11
+* Update CSI driver for Kubernetes 1.26
+  [[GH-469]](https://github.com/digitalocean/csi-digitalocean/pull/469)
```

Comment:
We may want to update the change log for the sidecar / controller updates now as well.

Reply:
@timoreimann , sure, I've added the new item referencing GH-747 and used [same message as v4.4.1](https://github.com/digitalocean/csi-digitalocean/blob/master/CHANGELOG.md#v441---20221104) for CHANGELOG item consistency.